packages/cysync-core/src/dialogs/Password/ChangePassword/Dialogs/CreateNewPassword.tsx

```diff
@@ -121,12 +125,14 @@ export const CreateNewPassword: React.FC = () => {
           type="submit"
           variant="primary"
           disabled={
+            loading ||
             Boolean(error) ||
             oldPassword.length === 0 ||
             newPassword.length < 8 ||
             confirmNewPassword.length < 8 ||
             newPassword !== confirmNewPassword
```

Comment:
All the validation checks should be in `context`

Reply:
validation checks moved to context